### ISSUE TITLE
Button not working when added after install

### ISSUE THREAD

**Issue by bernardogmorais** [cloudspout/cloudspout-button-panel#16]
I used the command to install

```
$ grafana-cli --pluginUrl https://github.com/cloudspout/cloudspout-button-panel/releases/download/7.0.2/cloudspout-button-panel.zip \
  plugins install cloudspout-button-panel

```

But when i try to use the plugin i have this error:
![image](https://user-images.githubusercontent.com/48386128/91334476-edf4b680-e7c6-11ea-8f33-96780abbf19d.png)

What's going on?

**Comment by derjust** [comment]
Can you post your Grafana (server) version please?

**Comment by derjust** [comment]
The issue should be fixed with the new release `7.0.5` of the plugin.
It is in review to be added to the Grafana web installer: https://github.com/grafana/grafana-plugin-repository/pull/756

Until then, a manual install via the CLI is already possible:
```BASH
$ grafana-cli --pluginUrl https://github.com/cloudspout/cloudspout-button-panel/releases/download/7.0.5/cloudspout-button-panel.zip \
  plugins install cloudspout-button-panel
```
